constants/Bingo.json

```
@@ -1,347 +1,351 @@
-        "Don't forget to talk to Guildford",
-        "after each run you do."
+        "You'll want to forge a Ruby Drill TX-15 (HotM4),",
+        "which needs 6 Fine Ruby Gems, a Fuel Canister (10h Forge Time)",
+        "and a Drill Motor (44h total Forge time). Once you gathered",
+        "all the materials, it needs 4h Forge Time."
       ]
     },
-    "Skill Average": {
-      "difficulty": "Medium",
+    "Fisherman's Folly": {
+      "difficulty": "Hard",
       "note": [
-        "Skill Average of 10"
+        "OUTSTANDING treasure/junk catch"
       ],
       "guide": [
-        "To help with this, try to make bigger crafts",
-        "by hand for Carpentry levels.",
-        "Use the Enchanted Sugar that you may get from",
-        "pests/the Garden to brew Speed Potions.",
-        "Also do your daily experiments and make sure",
-        "to have a pet active at all times",
-        "(any pet works, even level 100)."
+        "Recommended for this is any rod with the best",
+        "Fishing Speed you can get and Blessing 5+ Enchant.",
+        "If you want, you can further boost your chances",
+        "by using a Rare+ Hermit Crab Pet."
       ]
     },
-    "Healthy": {
-      "difficulty": "Medium",
+    "Chocolatier": {
+      "difficulty": "Easy",
       "note": [
-        "1250 Health"
+        "25m Chocolate"
       ],
       "guide": [
-        "Wear armor enchanted with Growth 5 ",
-        "(Enchanting Level 5), preferably Ender Armor",
-        "while you're in The End or Mushroom Armor",
-        "during the night.",
-        "Applying Growth 5 costs 50 Levels each.",
-        "If you still struggle, reforge your",
-        "armor to Titanic."
+        "Open the Chocolate Factory Menu with /cf,",
+        "get your first 50 Chocolate by clicking the button",
+        "in the middle of it and buy your first Level of",
+        "Rabbit Bro. Remember to check in every once in a while",
+        "and upgrade your workers. You can also grab",
+        "Chocolate Meal Eggs that spawn while Hoppity's Hunt is",
+        "around to further speed this up."
       ]
     },
-    "New Savings": {
+    "At your service": {
       "difficulty": "Medium",
       "note": [
-        "Craft a Piggy Bank"
+        "50 completed Garden Visitor offers"
       ],
       "guide": [
-        "This unlocks at 2.5k Raw Porkchop Collection",
-        "and requires 40 Enchanted Pork to make.",
-        "You can use Minions for this and/or pair",
-        "it with the Exterminator Grind if you do",
-        "that in Barn."
+        "Easy Visitors to go for are Jotraeline Greatforge",
+        "and Gold Forger as they don't ask for much each visit.",
+        "Other good choices to go for is the Librarian and any",
+        "visitor that wants Wheat like Jacob does.",
+        "(check the Wiki for more insight) Those can queue up to",
+        "6 visitors total at a time (5 queue + 1) without you",
```

```suggestion
        "5 visitors total at a time without you",
```
Last I checked the hidden 6th visitor didn't spawn offline.